### ISSUE TITLE
Color for time

### ISSUE THREAD

**Issue by ai** [andywer/postcss-debug#9]
We should highligh plugin and file time in different colors — green for the fastest and red to the slowest

/cc @Gaserd

**Comment by Gaserd** [comment]
@ai Yeap 👍

**Comment by andywer** [comment]
True. I also thought about that, but there are some detail issues:

Do we color by
- relative time (slowest/fastest for _current file_)
- relative time (slowest/fastest for _all files_)
- absolute time (let's say everything < 5ms is green, > 10ms is red, color gradient for 5ms < time < 10ms )
  ?

I guess a good solution would have to consider all three metrics, but only show one combined color.

**Comment by andywer** [comment]
PS: I just had an idea. I don't know if it's good or rather not, so I will just post it here:

We could ask the user for permission once and then sent simple anonymized benchmark reports to some server to gather statistics about the most popular postcss plugins and their performance benchmarks.

**Comment by Gaserd** [comment]
@andywer 
I think relative time (slowest/fastest for current file)
And collection statistics this good idea

**Comment by ai** [comment]
I think realtive time is fine too. Because the main purpose of coloring is to find slowest plugins and optimize them.

**Comment by 1j01** [comment]
There should probably be an absolute cutoff though, like less than 5 or 10 ms is always green, right?
